src/test/java/org/opensearch/knn/plugin/action/RestKNNStatsHandlerIT.java

```diff
@@ -36,7 +36,20 @@
 import static org.opensearch.knn.TestUtils.KNN_VECTOR;
 import static org.opensearch.knn.TestUtils.PROPERTIES;
 import static org.opensearch.knn.TestUtils.VECTOR_TYPE;
-import static org.opensearch.knn.common.KNNConstants.*;
```

Comment:
Pls avoid wildcards

Reply:
this line is removed, line 39-52 is the updated code.